python/tests/expressions/test_parser.py

```diff
@@ -149,3 +151,13 @@ def test_and_or_with_parens() -> None:
     assert Or(IsNull("x"), And(GreaterThanOrEqual("x", 5), Not(LessThan("x", 10)))) == parser.parse(
         "(x is null) or (5 <= x) and not(x < 10)"
     )
+
+
+def test_starts_with() -> None:
+    assert StartsWith("x", "data") == parser.parse("x starts_with 'data'")
```

Comment:
I don't think `starts_with` is common in SQL. How about:
```suggestion
    assert StartsWith("x", "data") == parser.parse("x LIKE 'data*'")
```

Reply:
Yes you are right about the fact that `start_with` is not common in SQL. I'm just afraid that `LIKE` is a little misleading because in SQL it's to look for a pattern in the whole string whereas here it would just be to look for "starts with".